File: list-tables.js
const { Client } = require('pg');

async function listTables() {
  const local = new Client({
    connectionString: 'postgresql://postgres:fulebimojviT1985%25@localhost:5432/ICE_ERP'
  });
  
  await local.connect();
  
  const result = await local.query(`
    SELECT table_name 
    FROM information_schema.tables 
    WHERE table_schema = 'public'
    ORDER BY table_name
  `);
  
  console.log('All tables:');
  result.rows.forEach(row => console.log(`- ${row.table_name}`));
  
  await local.end();
}

listTables().catch(console.error);

File: list-tables.py
import psycopg2, os
from dotenv import load_dotenv

load_dotenv('.env.vercel.production')
db_url = os.getenv('DATABASE_URL').split('?')[0]
conn = psycopg2.connect(db_url)
cur = conn.cursor()

cur.execute("""
    SELECT table_name 
    FROM information_schema.tables 
    WHERE table_schema = 'public' 
    AND (table_name LIKE '%duplicate%' OR table_name LIKE '%payment%')
    ORDER BY table_name
""")

print('Tables related to payments/duplicates:')
for row in cur.fetchall():
    print(f'  - {row[0]}')

conn.close()
